transcom/mymove
# Int b 18910 gun safe frontend

## [B-18910](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18910)

## Other INT PR
#12458

## Summary

This branch adds onto the other INT PR to make the gun safe entitlement visible to the Prime when they are viewing a move's information.

### How to test

1. Access the Move Details page of any move as an SC
2. Scroll down to the allowances card, verify that there is a new row at the bottom for gun safe, and that it is "Unauthorized".
3. Click the link, check the box for "Gun safe authorized", then click submit.
4. Scroll back down to the "Allowances" card and verify that the Gun Safe value is now "Authorized".
5. Repeat steps 1-4 while logged in as a TOO.
6. Now the move should be available to Prime, login as Prime and view the move, near the end of the move info, under "Diversion", there should now be a new field titled "Gun Safe", which should display the value you entered as SC/TOO.
